src/components/organisms/settings/organization/organization.tsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Controller, useForm, useWatch } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+
+import { ModalName } from "@src/enums/components";
+import { SelectOption } from "@src/interfaces/components";
+import { useModalStore } from "@src/store";
+import { organizationSchema } from "@validations";
+
+import { Button, ErrorMessage, Input, Typography } from "@components/atoms";
+import { Select } from "@components/molecules";
+import { DeleteOrganizationModal } from "@components/organisms/settings/organization";
+
+export const OrganizationSettings = () => {
+	const options: SelectOption[] = [
+		{ value: "disabled", label: "Disabled" },
+		{ value: "daily", label: "Daily" },
+		{ value: "hourly", label: "Hourly" },
+		{ value: "immediate", label: "Immediate" },
+	];
+
+	const { t } = useTranslation("settings", { keyPrefix: "organization" });
+	const {
+		control,
+		formState: { errors },
+		handleSubmit,
+		register,
+	} = useForm({
+		resolver: zodResolver(organizationSchema),
+		mode: "onSubmit",
+	});
+	const { openModal } = useModalStore();
+
+	const onSubmit = async () => {};
+
+	const watchedFrequency = useWatch({ control, name: "frequency" });
+
+	return (
+		<div>
+			<Typography className="mb-9 font-bold" element="h2" size="xl">
+				Organization {t("form.settings")}
+			</Typography>
+			<div className="flex flex-wrap gap-10">
+				<div className="w-1/2">
+					<Typography className="mb-2 font-bold" element="h2" size="medium">
+						{t("limits")}
+					</Typography>
+					<div className="flex gap-3">
+						<div className="w-1/2 rounded bg-gray-950 p-3 text-base shadow-xl">
+							<Typography className="mb-1" element="h5">
```

Comment:
The title should be in a higher order than the content.
For example, if the title is `h5`, then the content should be `h6` or better `p`.
```suggestion
							<Typography className="mb-1" element="h3">
```

Reply:
changed